### ISSUE TITLE
custom snippets not load

### ISSUE THREAD

**Issue by naaness** [Microsoft/vscode#22351]
- VSCode Version: Code 1.10.2 (8076a19fdcab7e1fc1707952d652f0bb6c6db331, 2017-03-08T14:00:46.854Z)
- OS Version: Linux x64 4.4.0-21-generic
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|crane|HvyIndustries|0.3.1|
|autofilename|JerryHong|0.1.3|
|php-formatter|Sophisticode|0.2.4|
|vscode-wakatime|WakaTime|1.1.3|
|alignment|annsk|0.3.0|
|vscode-eslint|dbaeumer|1.2.7|
|githistory|donjayamanne|0.2.0|
|theme-dracula|dracula-theme|1.8.0|
|auto-rename-tag|formulahendry|0.0.11|
|Theme-FlatlandMonokai|gerane|0.0.4|
|code-template-replace|heekei|0.1.2|
|volt|hunter-dev|0.0.1|
|phpcs|ikappas|0.7.0|
|php-symbols|linyang95|2.1.0|
|vs-deploy|mkloubert|5.34.1|
|lit-it|mohseenrm|1.2.10|
|vscode-icons|robertohuertasm|7.3.0|
|standard|shinnn|0.2.3|
|react-beautify|taichi|0.3.0|;

---

Steps to Reproduce:

1. Sometimes I try use snippets created by me, but its not load. I write the prefix and fail, press ctrl+space and fail

**Comment by ramya-rao-a** [comment]
Can you try by disabling the extension? Run `code --disable-extensions`
This is to ensure that the extensions are not interfering with expected behaviour

**Comment by naaness** [comment]
All extensions disabled but the same problem, look my snippets, is for javascript react
```javascript
{
	"Ajax": {
        "prefix": "_chronos_ajax",
        "body": [
            "Ajax.send({",
            "\turl:\"api/v1/${1:entity}/${2:action}\",",
            "\ttype:\"${3:GET}\",",
			"\tdata:{",
			"\t\t${4:data}",
			"\t},",
			"\tsuccess:this.${5:function}_success,",
			"\tsuccess_e:this.${5:function}_error,",
			"\terror:this.${5:function}_error",
            "});"
        ],
        "description": "Funcion ajax para chronos, captura la versión del sistema, y genera notificaciones automaticas cuando no existen las funciones de salida"
    },
	"Component class": {
        "prefix": "_chronos_component_class",
        "body": [ 
            "import React from \"react\";",
			"import Reflux from \"reflux\";",
			"import ${1:Action} from \"./${1:Action}.js\";",
			"import ${2:Store} from \"./${2:Store}.js\";",
            "",
            "var ${3:name} = React.createClass({",
			"\tmixins:[Reflux.connect(${2:Store},\"data\")],",
			"\tpropTypes: {",
			"\t\t${4}",
			"\t},",
			"\tgetDefaultProps() {",
			"\t\treturn {",
			"\t\t\t${5}",
			"\t\t};",
			"\t},",
			"\tgetInitialState: function() {",
			"\t\tlet data = {${6}};",
			"\t\t${1:Action}.setData(data);",
			"\t\treturn {data:data};",
			"\t},",
			"\tcomponentWillUnmount:function() {",
			"\t\t${1:Action}.destroyCallbacks();",
			"\t},",
			"\trender: function(){",
			"\t\tlet d= this.state.data;",
			"\t\tlet p= this.props;",
			"\t\tif(!d) return null;",
			"\t\treturn (",
			"\t\t\t<div>${7}</div>",
			"\t\t);",
			"\t}",
            "});",
			"export default ${3:name};"
        ],
        "description": "Componente usando class Acciones y Stores"
    },
	"Store":{
		"prefix": "_chronos_store",
		"body": [
			"import Reflux from \"reflux\";",
			"import ${1:Action} from \"./${1:Action}\";",
			"import PS from \"../../stores/PS.js\";",
			"",
			"let ${2:Store} = Reflux.createStore({",
			"\tlistenables: [${1:Action}],",
			"\tdata: null,",
			"\tsetData: function (data) {",
			"\t\tthis.data = data;",
			"\t\tthis.registryCallbacks();",
			"\t},",
			"\tregistryCallbacks:function () {",
			"\t\tif( typeof this.tokensPS == \"undefined\"){this.tokensPS = [];}else{return;}",
			"\t\tthis.tokensPS.push(PS.subscriber(\"${3:watcher}\",this.watcher_${3:watcher}));",
			"\t},",
			"\twatcher_${3:watcher}:function (topic,args){",
			"\t\t",
			"\t},",
			"\tdestroyCallbacks:function () {",
			"\t\tfor (let i=0; i < this.tokensPS.length; i++){PS.unsubscribe(this.tokensPS[i]);}",
			"\t\tdelete this[\"tokensPS\"];",
			"\t}",
			"});",
			"export default ${2:Store};"
		],
		"description": "Store conectado a la accion"
	},
	"Action":{
		"prefix": "_chronos_action",
		"body": [
			"import Reflux from \"reflux\";",
			"",
			"let ${0:Action} = Reflux.createActions([",
			"\t\"setData\",",
			"\t\"destroyCallbacks\",",
			"]);",
			"",
			"export default ${0:Action};"
		],
		"description": "Acciones del componente"
	},
	"Component": {
        "prefix": "_chronos_component",
        "body": [
            "import React from \"react\";",
			"import PS from \"../../stores/PS.js\";",
            "",
			"class ${1:name} extends React.Component{",
			"\tcomponentWillUnmount () {",
			"\t\tfor (let i=0; i < this.tokensPS.length; i++){PS.unsubscribe(this.tokensPS[i]);}",
			"\t\tdelete this[\"tokensPS\"];",
			"\t}",
			"\tconstructor(props) {",
			"\t\tsuper(props);",
			"\t\tthis.state = {${2}};",
			"\t\tif( typeof this.tokensPS == \"undefined\"){this.tokensPS = [];}",
			"\t\tthis.tokensPS.push(PS.subscriber(\"${3:watcher}\",this.watcher_${3:watcher}.bind(this)));",
			"\t}",
			"\twatcher_${3:watcher}(topic,args){",
			"\t\t",
			"\t}",
			"\trender(){",
			"\t\tlet p = this.props;",
			"\t\tlet d = this.state;",
			"\t\treturn (",
			"\t\t\t<div>${4}</div>",
			"\t\t)",
			"\t}",
			"}",
			"export default ${1:name};",
			"${1:name}.propTypes = {",
			"\t${5}",
			"};",
			"${1:name}.defaultProps = {",
			"\t${5}",
			"};$0"
        ],
        "description": "Componente con PS"
    },
	"Function Success Error": {
        "prefix": "_chronos_f_success_error",
        "body": [
			"${1:function}_success:function(result){",
			"\t",
			"},",
			"${1:function}_error:function(data){",
			"\t",
			"},"
		],
		"description": "Crea por defecto las funciones de exito y error"
	},
	"Function Get Success Error": {
        "prefix": "_chronos_f_get_ajax_success_error",
        "body": [
			"${1:function}:function(${2:page}){",
			"\tAjax.send({",
			"\t\turl:\"api/v1/${3:entity}/${4:action}\",",
			"\t\ttype:\"${5:GET}\",",
			"\t\tdata:{",
			"\t\t\t${2:page}:${2:page}",
			"\t\t},",
			"\t\tsuccess:this.${1:function}_success,",
			"\t\tsuccess_e:this.${1:function}_error,",
			"\t\terror:this.${1:function}_error",
			"\t});",
			"},",
			"${1:function}_success:function(result){",
			"\t",
			"},",
			"${1:function}_error:function(data){",
			"\t",
			"},"			
		],
		"description": "Crea por defecto las funciones de get con Ajax y las funciones de exito y error"
	}
}
```

**Comment by jrieken** [comment]
In what file and where in that file are you trying to insert a snippet? Does the snippet show when you go via 'F1 > Insert Snippet'. Does it show, not show in the IntelliSense box via Ctrl+Space?

**Comment by naaness** [comment]
@jrieken the snippet does not show when I go 'F1 > Inser Snippet', And when I press 'Crtl+Space' it does not show. I am using Linux MInt

**Comment by javorosas** [comment]
Same for me. Custom snippets are not showing in Ubuntu 16.04, VSCode 1.10.2

**Comment by sot528** [comment]
FWIW.

I had same problem via `Tab` key, not `Ctrl+Space`.
In my case, it improved when turn `true` the setting `editor.tabCompletion`.

**Comment by xiavn** [comment]
I noticed you mentioned you saved it as a Javascript React snippet - I seemed to have this same issue, I couldn't get the snippet to show up at all, not in IntelliSense or Insert Snippet. As soon as I saved the snippet to Javascript (rather than Javascript React) it showed up. I tried both options, with the files language settings set as both Javascript or Javscript React, and it didn't matter what the file was set to, snippets only showed up when saved in the javascript snippets file.

I'm not sure what benefit setting the language to Javascript React actually has?
